ruby/example_code/ec2/ec2-ruby-example-create-subnet.rb

```diff
@@ -1,36 +1,113 @@
-# snippet-comment:[These are tags for the AWS doc team's sample catalog. Do not remove.]
-# snippet-sourceauthor:[Doug-AWS]
-# snippet-sourcedescription:[Creates a public subnet with tags and attaches it to a VPC.]
-# snippet-keyword:[Amazon Elastic Compute Cloud]
-# snippet-keyword:[create_subnet method]
-# snippet-keyword:[Subnet.create_tags method]
-# snippet-keyword:[Ruby]
-# snippet-sourcesyntax:[ruby]
-# snippet-service:[ec2]
-# snippet-keyword:[Code Sample]
-# snippet-sourcetype:[full-example]
-# snippet-sourcedate:[2018-03-16]
-# Copyright 2010-2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX - License - Identifier: Apache - 2.0
+
+require 'aws-sdk-ec2'
+
+# Creates a subnet within a virtual private cloud (VPC) in 
+#   Amazon Virtual Private Cloud (Amazon VPC) and then tags

```

Comment:
Why the extra initial spaces?

Reply:
It seems in the Ruby YARD spec there are examples that show both initial spaces and no initial spaces for descriptions. However, there do seem to be more examples that show no initial spaces, so I'll remove them.